frontend/pages/DashboardPage/cards/ActivityFeed/components/ScriptDetailsModal/ScriptDetailsModal.tsx

```diff
@@ -158,14 +158,18 @@ const ScriptResult = ({
 };
 
 interface IScriptDetailsModalProps {
+  scriptExecutionId: string;
   onCancel: () => void;
 }
 
-const ScriptDetailsModal = ({ onCancel }: IScriptDetailsModalProps) => {
+const ScriptDetailsModal = ({
+  scriptExecutionId,
+  onCancel,
+}: IScriptDetailsModalProps) => {
   const { data, isLoading, isError } = useQuery<IScriptResult>(
     ["scriptDetailsModal"],
```

Comment:
```suggestion
    ["scriptDetailsModal", scriptExecutionId],
```

Reply:
great catch, I think it's important we do this, right? I will follow up right after this is merged (so I don't lose the other ✅ and the CI status)